host/4/republish.yml

```diff
@@ -62,6 +62,16 @@ steps:
     displayName: tag and push dotnet images
     continueOnError: false
 
+  - bash: |
+      set -e
+      docker pull $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet8
+      docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet8 $TARGET_REGISTRY/dotnet:$(TargetVersion)-dotnet8
+      docker push $TARGET_REGISTRY/dotnet:$(TargetVersion)-dotnet8
```

Comment:
Unfortunately this does not include the -appservice image. 


This pipeline is the one that produces floating image tags like :4-dotnet8-appservice. We need to pull the image, tag it and push it